tests/unit/test_adapter.py

```diff
@@ -1,153 +1,41 @@
-                        "port": 10001,
-                        "user": "dbt",
-                    }
-                },
-                "target": "test",
-            },
-        )
-
-    def _get_target_thrift_kerberos(self, project):
-        return config_from_parts_or_dicts(
-            project,
-            {
-                "outputs": {
-                    "test": {
-                        "type": "spark",
-                        "method": "thrift",
-                        "schema": "analytics",
-                        "host": "myorg.sparkhost.com",
-                        "port": 10001,
-                        "user": "dbt",
-                        "auth": "KERBEROS",
-                        "kerberos_service_name": "hive",
-                    }
-                },
-                "target": "test",
-            },
-        )
-
-    def _get_target_use_ssl_thrift(self, project):
-        return config_from_parts_or_dicts(
-            project,
-            {
-                "outputs": {
-                    "test": {
-                        "type": "spark",
-                        "method": "thrift",
-                        "use_ssl": True,
-                        "schema": "analytics",
-                        "host": "myorg.sparkhost.com",
-                        "port": 10001,
-                        "user": "dbt",
-                    }
-                },
-                "target": "test",
-            },
-        )
-
-    def _get_target_odbc_cluster(self, project):
-        return config_from_parts_or_dicts(
-            project,
-            {
-                "outputs": {
-                    "test": {
-                        "type": "spark",
-                        "method": "odbc",
-                        "schema": "analytics",
-                        "host": "myorg.sparkhost.com",
-                        "port": 443,
-                        "token": "abc123",
-                        "organization": "0123456789",
-                        "cluster": "01234-23423-coffeetime",
-                        "driver": "Simba",
-                    }
-                },
-                "target": "test",
-            },
-        )
-
-    def _get_target_odbc_sql_endpoint(self, project):
-        return config_from_parts_or_dicts(
-            project,
-            {
-                "outputs": {
-                    "test": {
-                        "type": "spark",
-                        "method": "odbc",
-                        "schema": "analytics",
-                        "host": "myorg.sparkhost.com",
-                        "port": 443,
-                        "token": "abc123",
-                        "endpoint": "012342342393920a",
-                        "driver": "Simba",
-                    }
-                },
-                "target": "test",
-            },
-        )
+class TestSparkAdapter(unittest.TestCase):
+    @pytest.fixture(autouse=True)
+    def set_up_fixtures(
```

Comment:
Since `pytest` fixtures are globally available, is this necessary? For example, wouldn't `test_http_connection` just be:

```python
def test_http_connection(self, target_http):
    adapter = SparkAdapter(target_http, get_context("spawn"))
    ...
```

Reply:
I tried that but since this is a `unittest.TestCase` it doesn't work out of the box, I would like to refactor these to not be class based/rely on unittest but figured that would kind of blow up the PR

Reply:
Ah I missed that, makes sense.